### DISCUSSION TITLE
Map icons representing users

### DISCUSSION THREAD

**Discussion by ahillbilly** [Pray4Movement/prayer-global-porch#27] [Ideas]
The current map shows all the prayer warriors with a common single image replicated where ever the warrior is located.  

What would it take (when there are user accounts) for a user to set their location, pick their image and that image show up on the map to represent them in any lap that they've participated.  

I think this could contribute to user engagement and the person seeing themself clearly as a part of a much bigger community.  It would add a fun factor and differentiate the prayer warriors mapped.

![Screen Shot 2022-10-04 at 11 01 55 AM](https://user-images.githubusercontent.com/25323324/193792150-96ca11b6-53a2-4552-9de6-db29ca5ca513.png)

**Comment by prykon** [comment]
I really like this idea! Here are some comments...

User avatars could be uploaded, selected from a generic avatar gallery with fun Bible-related icons and other misc icons and the default avatar for no avatar can maybe be one like the Facebook profile silhouette guy.

Some gallery avatar ideas can be:
- Icthys
- Bread and cup
- Cross
- White dove
- Fire
- Lion
- Flags for each country
- etc.

Avatars should all have the same look & feel.
